airflow/executors/base_executor.py

```diff
@@ -128,7 +128,9 @@ def heartbeat(self):
             ti.refresh_from_db()
             if ti.state != State.RUNNING:
                 self.running[key] = command
-                self.execute_async(key, command=command, queue=queue)
+                uuid = self.execute_async(key, command=command, queue=queue)
+                if uuid:
```

Comment:
I like the idea of keeping track of the `uuid`'s. Just curious, are there any situations when you get a null?

Reply:
@Fokko if the executor doesn't return anything, The uuid will be a None, and shows up empty in the webserver column etc.